cognite/client/_cognite_client.py

```diff
@@ -123,6 +124,7 @@ def __init__(
             self._config, api_version="playground", cognite_client=self
         )
         self.transformations = TransformationsAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
+        self.annotations = AnnotationsAPI(self._config, api_version="playground", cognite_client=self)
```

Comment:
TODO: update `api_version` once the endpoints have been migrated to V1

Reply:
[97f72ef](https://github.com/cognitedata/cognite-sdk-python/pull/928/commits/97f72efaf185fe073a6ed3c3e28b1cc72afe8a02)